recipes/get_orfs/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "get_orfs" %}
-{% set version = "1.0.0" %}
+{% set version = "0.2.0" %}
```

Comment:
```
This is bumping the version that adds minor functionality.
```
Actually it is a downgrade.
I see 0.1.0 and 0.2.0 at https://github.com/linsalrob/get_orfs/releases. I am not sure how 1.0.0 worked before ...

Reply:
v1.0.0 was a tag https://github.com/linsalrob/get_orfs/tags

Though chronologically, it was released befire 0.1.0 ?!?

Reply:
I made a right mess of the versioning here, and so I've bumped the latest version to 1.1.0 to overcome this mess. Apologies all around.